cli/azd/pkg/alpha/alpha_features.go

```diff
@@ -2,6 +2,7 @@ package alpha
 
 const (
 	TerraformId FeatureId = "terraform"
+	BicepADS    FeatureId = "bicep.ads"
```

Comment:
Just wondering - do we want to put this behind a feature flag?  It feels like the sort of performance improvement we might just normally do.  I understand that we could be worried because this is some new thing we are doing, but the user can always pass `--no-cache` to opt out of this behavior correct?

Reply:
Yes.
I am happy to remove it from `alpha` !!  It would make it even easier then.

Somehow, in my mind, I was thinking that anything new for azd should go through the feature-graduation process as an introduction process for customers.

But this can be seen as an add-on to the current bicep-provisioning model, and not really like something new.